@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -870,8 +888,19 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
                             geom.faces.push(
                                 new THREE.Face3(triangles[i], triangles[i + 1], triangles[i + 2])
                             );
+                            if (texCoordType !== undefined) {
+                                const i0 = triangles[i] * stride;
```

Comment:
why do you * by stride?

Reply:
b/c triangles contains the vertex index but vertices is not an array of structs but a plain number[].